### ISSUE TITLE
Export image source (editable version of image

### ISSUE THREAD

**Issue by vtlkzmn** [shutter-project/shutter#345]
Hello, I'm very new to the tool, but already found it very useful, thank you so much!

I faced the issue when I added some text to the picture and then saved it, edit window closed and after I reopened it I do not see anymore options to edit text I have entered in the beginning. Seems like I have saved an image of the edited object. Okay, my fault, no problem, but I do not see any option to export fully editable source image with all the possibilities to select text, arrows etc after saving/exporting to any of observable options. Briefly scrolling web page/google didn't find an answer. So probably it's feature request.
Thanks in advance!

**Comment by Photon89** [comment]
It is indeed, there is no such functionality yet!

**Comment by DarthGandalf** [comment]
Maybe at this point you should use a tool such as [gimp](https://www.gimp.org/) or [krita](https://krita.org)?

Saving such source means inventing an image format to store all the objects added to the image, which is a poor interop with any other image editing software. What's worse, we would need to care about back compatibility of this format between versions of shutter, making fixes of issues such as #343 harder, because behavior of tools can change over time, and without supporting both behaviors the image you loaded in new version is not the same image you saved in old version.

**Comment by vadi2** [comment]
I agree - it's good to draw the line here to ease maintenance load. There's only so much manpower.

**Comment by Photon89** [comment]
Agreed. So, that's a "won't fix", I guess?

**Comment by DarthGandalf** [comment]
Perhaps unless someone implements it?

**Comment by vadi2** [comment]
Let's say out of scope.

**Comment by kuhnchris** [comment]
*technically* one could go and simply "store the undo buffer" and the original image, and upon processing simply execute (/redo) the buffer stored in a separate file.
still would require someone to implement it tho.

**Comment by Photon89** [comment]
In the mean time I figured out that the SVG export in the drawing tool does exactly what is requested here, see also https://github.com/shutter-project/shutter/issues/449#issuecomment-1213809119 Only the undo history is not saved this way, but the elements added in the drawing tool remain editable. What is still missing though is the SVG import, so the elements can only be further edited in tools like Inkscape, not in the drawing tool itself.
